exercises/practice/series/.meta/Example.roc

```diff
@@ -0,0 +1,25 @@
+module [slices]
+
+slices : Str, U64 -> Result (List Str) [StringArg [WasEmpty, WasNotAscii Str], SliceLengthArg [WasZero, WasTooLarge U64 Str]]
+slices = \string, sliceLength ->
+    chars = string |> Str.toUtf8
+    len = chars |> List.len
+    if len == 0 then
+        StringArg WasEmpty |> Err
+    else if sliceLength == 0 then
+        SliceLengthArg WasZero |> Err
+    else if sliceLength > len then
+        SliceLengthArg (WasTooLarge sliceLength string) |> Err
+    else
+        maybeResult =
+            List.range { start: At 0, end: At (len - sliceLength) }
+            |> List.map \startIndex ->
+                chars
+                |> List.sublist { start: startIndex, len: sliceLength }
+                |> Str.fromUtf8
```

Comment:
You could use List.mapTry here which will give you back a Result (List a) err instead of a List (Result a err)

Reply:
Oh I did not know about `List.mapTry`, that's so much better, thanks, I'll fix this now

Reply:
Yeah, I was very pleased when I saw mapTry also, it's quite nice 😃